### PR TITLE
ICDS-CAS: changes in date filtering

### PR DESCRIPTION
**Pull request by lwyszomi**
@sravfeyn

### CODE REVIEW

File: custom/icds_reports/reports/__init__.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def report_config(self):
         start_date, end_date = get_first_last_days(self.year, self.month)
         month_before = start_date - relativedelta(months=1)
         new_start_date = month_before.replace(day=21)
-        new_end_date = month_before.replace(day=20)
+        new_end_date = start_date.replace(day=20)
```
Comment (sravfeyn) [inline custom/icds_reports/reports/__init__.py]: @sheelio Do you have any context on filtering for MPR report?

**Comment by lwyszomi** [comment]
@NoahCarnahan Do you have any context to these reports? 

http://manage.dimagi.com/default.asp?249823#1308767

**Comment by sheelio** [comment]
@sravfeyn 

I have context on the filtering - it should be on the 21st of previous month to 20th of the current month when you choose a month. 

A longish spec is here:
https://docs.google.com/a/dimagi.com/document/d/11vpRqLinKYsPq7w3KWNqGTHEitjas321_p7gF208rsU/edit?usp=drive_web

**Comment by NoahCarnahan** [comment]
@lwyszomi I do not (Sounds like Sheel has you covered though).

**Comment by sravfeyn** [comment]
@lwyszomi I have approved the PR, feel free to self-merge if you have know the needed behaviour

**Review by sravfeyn — APPROVED** [review]

**Comment by lwyszomi** [comment]
@sravfeyn I don't have access to merge this.

**Pull request merged**